RELEASE_NOTES.md

```diff
@@ -1,3 +1,95 @@
+## Cookstyle 5.10
+
+### 9 New Chef Cops
+
+#### ChefDeprecations/VerifyPropertyUsesFileExpansion
+
+The `VerifyPropertyUsesFileExpansion` cop detects resources using the verify property to validate file content with the legacy `file` variable instead of the `path` variable required in Chef Infra Client 13 and later.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefDeprecations/ResourceInheritsFromCompatResource
+
+The `ResourceInheritsFromCompatResource` cop detects legacy HWRPs that inherit from the CompatResource class that shipped in the legacy `compat_resource` cookbook. Ideally these resources are rewritten as standard Custom Resources, but at a minimum they should inherit from the proper class: `Chef::Resource`.
```

Comment:
What are HWRPs? (I'm not sure if that should be abbreviated or not.)

Reply:
Heavy Weight Resource Provider

Reply:
I expanded this